vtex-apps/checkout-shipping
# CHK-568: Fix pickup modal displaying wrong address

#### What problem is this solving?

The pickup modal is currently displaying the user own address instead of the store address.

#### How should this be manually tested?

[Workspace](https://wesley--checkoutio.myvtex.com/cart)

1. Add [this product](https://wesley--checkoutio.myvtex.com/teste-so-delivery-copy-256--copy-257--copy-258-/p)
2. Use 22250-040 as postal code
3. Select "Loja em Copacabana no Rio de Janeiro"
4. Click "Ver detalhes"
5. It should display the store address (like in the screenshot)

#### Screenshots or example usage

Before | After
-|-
![checkoutio myvtex com_cart](https://user-images.githubusercontent.com/26108090/109982809-a2244c00-7cd8-11eb-9a8b-1005c86ce007.png) | ![wesley--checkoutio myvtex com_cart](https://user-images.githubusercontent.com/26108090/109979080-e1e93480-7cd4-11eb-89f5-1288a9d08816.png)
